draft-ietf-rats-architecture.md

```diff
@@ -375,16 +375,20 @@ A Verifier uses the Evidence, any Reference Values from Reference Value Provider
 by applying an Appraisal Policy for Evidence to assess the trustworthiness of the Attester.
 This procedure is called the appraisal of Evidence.
 Subsequently, the Verifier generates Attestation Results for use by Relying Parties.
-The Appraisal Policy for Evidence
-might be obtained from an Endorser along with the Endorsements, and/or might be obtained
-via some other mechanism, such as being configured in the Verifier by the Verifier Owner.
+
+The Appraisal Policy for Evidence might be obtained from the Verifier Owner,
+and/or is configured into the Verifier by the Verifier Owner,
+and/or is programmed into the Verifier,
+and/or might be obtained via some other mechanism.
```

Comment:
So, if I choose "and" at every decision-point, is that statement still true?

Reply:
I think just "or" would be better, to Henk's point.